packages/create-novu-app/index.ts

```diff
@@ -19,6 +19,8 @@ let projectPath = '';
 
 const handleSigTerm = () => process.exit(0);
 
+const defaultTunnelHost = 'https://localtunnel.me';
```

Comment:
for some reason the default value in the option doesn't work so I needed to place it here to have access to it.